coordinator/points_writer.go

```diff
@@ -121,32 +124,18 @@ func (s *ShardMapping) MapPoint(shardInfo *meta.ShardInfo, p models.Point) {
 
 // Open opens the communication channel with the point writer.
 func (w *PointsWriter) Open() error {
-	w.mu.Lock()
-	defer w.mu.Unlock()
 	w.closing = make(chan struct{})
 	return nil
 }
 
 // Close closes the communication channel with the point writer.
 func (w *PointsWriter) Close() error {
-	w.mu.Lock()
-	defer w.mu.Unlock()
```

Comment:
Was this causing problems, or is this cleanup of unneeded code?

Reply:
Cleanup of unneeded code. I don't believe we can actually call Open or Close concurrently at the moment (they are set up and torn down by the Server). This lock in theory locked subPoints, but we no longer need the lock in WritePointsPrivileged.